ceph/ceph.py

```diff
@@ -47,6 +47,7 @@ def __init__(self, name, node_list=None):
         self.custom_config = None
```

Comment:
this change should go here - https://github.com/red-hat-storage/cephci/pull/3157